internal/service/project/data_source_projects.go

```diff
@@ -143,22 +142,17 @@ func (d *ProjectsDS) Schema(ctx context.Context, req datasource.SchemaRequest, r
 
 func (d *ProjectsDS) Read(ctx context.Context, req datasource.ReadRequest, resp *datasource.ReadResponse) {
 	var stateModel tfProjectsDSModel
-	conn := d.Client.Atlas
 	connV2 := d.Client.AtlasV2
 
 	resp.Diagnostics.Append(req.Config.Get(ctx, &stateModel)...)
-	options := &matlas.ListOptions{
-		PageNum:      int(stateModel.PageNum.ValueInt64()),
-		ItemsPerPage: int(stateModel.ItemsPerPage.ValueInt64()),
-	}
```

Comment:
any reason to ignore pagination now?

Reply:
With new SDK, `ListProjects` uses default values for pagination. Fixed it so I don't change the behaviour. Thanks!

Reply:
the old SDK should've had defaults as well, reason to ask is I have no context where `stateModel` was and if those values where meaningful